css/DesktopVersion.css

```diff
@@ -0,0 +1,417 @@
+
+  main {
+    display: grid;
+    grid-template-rows: repeat(1fr);
+    justify-content: center;
+    align-items: center;
+    height: 900px;
+    background: white;
+    background-image: url(../DeskImg/Header\ bg@2x.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-bottom-left-radius: 200px;
+  }
+
+  .titleText .beConnect {
+    width: 217px;
+    height: 24px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .titleText h1 {
+    width: 920px;
+    height: 122px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    margin-top: 30px;
+  }
+
+  .titleText p {
+    margin-top: 50px;
+    margin-bottom: 50px;
+    width: 920px;
+    height: 98px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .LinkNetwork ul li {
+    display: flex;
+    justify-content: flex-start;
+    position: relative;
+    left: -59px;
+    margin: 10px;
+  }
+
+  .LinkNetwork ul {
+    display: flex;
+    justify-content: flex-start;
+    list-style: none;
+    flex-wrap: nowrap;
+    margin: 0;
+
+    /* new */
+  }
+
+  /* none displayer session */
+  .imgOff {
+    display: none;
+  }
+
+  /* Trying design */
+  .template {
+    display: grid;
+    justify-content: center;
+    max-width: 80%;
+    margin: 0 auto;
+    grid-template-columns: repeat(2, 1fr);
+    background-color: #fff;
+    border-radius: 24px;
+    gap: 20px;
+    margin-top: 100px;
+    padding: 20px;
+  }
+
+  .template img {
+    width: 100%;
+  }
+
+  .template h1 {
+    font-size: 24px;
+    font-weight: 700;
+    color: #091e42;
+  }
+
+  .template .tempUl {
+    display: flex;
+    list-style-type: disc !important;
+    align-items: left;
+    flex-wrap: wrap;
+  }
+
+  .template .tempUl li {
+    margin-right: 27px;
+    color: #6b778c;
+    list-style-type: disc !important;
+    font-size: 18px;
+    font-weight: 400;
+  }
+
+  .template .tempUl li:first-child {
+    color: #344563;
+    font-weight: 500;
+    margin-left: 0;
+  }
+
+  .template .tempLang {
+    display: flex;
+    list-style-type: none;
+    flex-wrap: wrap;
+  }
+
+  .template .tempLang .luga {
+    font-size: 12px;
+    padding: 5px 5px;
+    color: #6070ff;
+    border-radius: 8px;
+    background: #ebebff;
+    margin-right: 10px;
+  }
+
+  .template p {
+    width: 496px;
+    height: 50px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .about-me {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: repeat(2, 1fr);
+    background-color: #fff;
+    border-top-right-radius: 80px;
+    gap: 20px;
+    padding: 80px;
+    margin-top: 50px;
+  }
```

Comment:
- [ ] Good work building your Desktop About me section 💯. However, please kindly ensure to fix the margins, and element sizes in your design to stick as much as possible to the Figma design as requested by the previous reviewer. 🤓 Also, please kindly ensure to fix the typo in your Get my resume button. 🔧

This is the Figma design 👇🏽

![figma](https://user-images.githubusercontent.com/116701485/236357424-29a75976-c000-4b31-b4a3-6046d6878140.png)

This is your design 👇🏽

![design](https://user-images.githubusercontent.com/116701485/236357445-5294011d-8540-4624-b5f4-8a099e36ea6e.png)